scripts/processing_csv.py

```diff
@@ -0,0 +1,89 @@
+import csv
+import json
+import re
+
+
+# Converts a list of dictionaries (assumed to be JSON data) into CSV format.
+def json_to_csv_data(json_data):
+    if not isinstance(json_data, list):
+        raise ValueError("JSON data is not a list of dictionaries")
+    csv_data = []
+    fieldnames = json_data[0].keys()
+    for item in json_data:
+        csv_data.append({fieldname: item.get(fieldname, "") for fieldname in fieldnames})
+    return csv_data, fieldnames
```

Comment:
this returns a list of dictionaries right? then it is unclear to me why you call it `csv_data`?

Reply:
yes dictionaries, this is just a name I gave as these data will convert to CSV later, I'm changing them to `
dictionaries_data`

Reply:
ok, sounds good.